### DISCUSSION TITLE
Some thalamus items in wrecks are marked hidden in-game

### DISCUSSION THREAD

**Discussion by aoyamajia** [FakeFishGames/Barotrauma#14171] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The following wrecks have some or all cellspawnorgans/fleshgunammosacks/storageorgans marked hidden:
- Berilia_Wrecked
- Dugong_Wrecked
- Hemulen_Wrecked
- Kastrull_Wrecked
- Wreck1

I have checked the other wrecks not listed and they do not seem to be affected.

![image](https://github.com/FakeFishGames/Barotrauma/assets/39614559/fa059c7f-e3da-44f7-9a8a-3706da49d65c)
![image](https://github.com/FakeFishGames/Barotrauma/assets/39614559/b4e4f544-4356-4e72-8034-d917b83d7409)

### Reproduction steps

1. Open any of the above wrecks and notice how the mentioned thalamus items are marked hidden in-game.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by PanmanS** [comment]
So True bestie

**Comment by Jasontti** [comment]
Tested the fix and found no problems.
Closing discussion as resolved.
